bot/src/main/resources/translations/horo/profile.properties

```diff
@@ -0,0 +1,13 @@
+tag.arguments.user.name=user
+tag.arguments.user.description=tag a user
+name=profile
+description=Profile commands
+avatar.profile.sub.name=avatar
+avatar.profile.sub.description=Display a user's avatar
+avatar.source.profile.sub.button=Open source
+view.profile.sub.name=view
+view.profile.sub.description=Display a user's avatar
+view.profile.sub.field.nickname=Nickname
+view.profile.sub.field.joinedAt=Joined at
+view.profile.sub.field.createdAt=Created at
+view.profile.sub.field.roles=Roles
```

Comment:
This file is missing a newline at the end, which will mess up Git diffs that modify the end of the file.